demos/blog/wait-for-postgres.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# from https://docs.docker.com/compose/startup-order/
+# wait-for-postgres.sh
```

Comment:
This seems like the wrong approach. As the doc linked above says, it's better to make the application robust against a not-yet-started database than to wait before starting the app (at the very least, an error page served by blog.py would be preferable to the "connection refused" that would result here). I think we should figure out how to refactor blog.py to make this work correctly.

Reply:
Sure. What if we close this PR, then open an issue explaining the problem, with a link to this closed PR, so that either myself or somebody else will tackle it? I will have some spare time again in a couple weeks I think (but happy if another contributor decides to work on it 👍 )

Reply:
Sounds good.

Reply:
Done https://github.com/tornadoweb/tornado/issues/2788